assets/src/design-system/components/tab/index.js

```diff
@@ -32,16 +32,16 @@ const StyledTab = styled.button(
     padding: 0 16px;
 
     background-color: ${isActive
-      ? theme.colors.bg.primary
-      : theme.colors.bg.divider};
+      ? theme.colors.interactiveBg.primaryNormal
+      : theme.colors.opacity.footprint};
     border: none;
     border-radius: 50px;
     ${themeHelpers.focusableOutlineCSS(
       theme.colors.bg.primary,
-      theme.colors.accent.secondary
+      theme.colors.border.focus
     )};
 
-    color: ${isActive ? theme.colors.fg.primary : theme.colors.fg.secondary};
+    color: ${isActive ? theme.colors.bg.primary : theme.colors.fg.primary};
```

Comment:
The inactive dark theme color is brighter here than the figma pill.

Reply:
good catch, this should actually be `fg.secondary` afterall.